### PR TITLE
setup listeners: use capture mode

### LINKED ISSUE #3398: Some keys do not work on https://www.nydailynews.com/ (both chrome and firefox)
link hints (f/F), open URL (o/O) keys don't work
some keys like scrolling, tab navigation etc., do work.

chrome: 77.0.3865.75
firefox: 69.0
Vimium: 1.64.6

Regards

### PR DESCRIPTION
**Pull request by gdh1995**
This should fix #3398 (the problematic page suppresses DOMContentLoaded events).

### CODE REVIEW

File: content_scripts/ui_component.coffee [hunk 1]
```diff
@@ -62,10 +62,11 @@ class UIComponent
                       @hide false
                     false # We will not be calling sendResponse.
                   # If this frame receives the focus, then hide the UI component.
-                  window.addEventListener "focus", (event) =>
+                  window.addEventListener "focus", ((event) =>
                     if event.target == window and @options?.focus
                       @hide false
-                    true # Continue propagating the event.
+                    true
+                  ), true # Continue propagating the event.
```
Comment (lydell) [inline content_scripts/ui_component.coffee]: The comment belongs to the `true` on the line above, doesn’t it?

File: pages/vomnibar.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
   <body>
     <div id="vomnibar" class="vimiumReset">
       <div class="vimiumReset vomnibarSearchArea">
-        <input id="vomnibarInput" type="text" class="vimiumReset">
+        <input id="vomnibarInput" type="text" class="vimiumReset" autocomplete="off">
```
Comment (lydell) [inline pages/vomnibar.html]: Looks unrelated to the rest of the PR?

Comment (gdh1995) [reply pages/vomnibar.html]: yes, unrelated😂

File: content_scripts/ui_component.coffee (hunk 1 above)
Comment (gdh1995) [reply content_scripts/ui_component.coffee]: updated.

in fact, the first 'true' here does nothing - Browser doesn't use true/false to decide whether to continue on such listeners

**Comment by philc** [comment]
It seems conceptually reasonable that we would listen to the capture phase of an event, and this should make Vimium more robust to individual pages suppressing important events.

It's a pretty core change, however. Have you foreseen any issues yet with this?

**Comment by gdh1995** [comment]
Compared with bubble-phase listeners, the only obvious differences is, for `focus`, `blur`, `load`, and `unload` events:
* capture-phase listeners will be run if only a bound target or its descendant generates such a event
* but bubble-phase listeners will only accept events whose `target` is the bound `this`

Maybe there're also some other events following such a rule, but it will always be safe if we add enough checks.

My Vimium C has used caputure-phase listeners for 4 years and no unexpected breaks found.

**Pull request merged**